config/nginx/conf.d/nginx.conf

```diff
@@ -8,6 +8,7 @@ server {
     server_name monitoring.localzero.net monitoring-test.localzero.net;
 
     client_max_body_size 100m;
+    add_header Content-Security-Policy "default-src 'self'";
```

Comment:
this disallows adding images from external sources. I am not sure whether we want this or not, one can set img-src to something different if needed.

Reply:
External images might be useful for graphics that change regularly (e.g. a solar power chart Munich gets from the Frankfurt team who runs an app that updates it regularly).
I'm not sure how much of an use case this is... We might just disallow that for now and see if someone complains..?